index.js

```diff
@@ -20,10 +20,28 @@ program
   .option('-p, --profile <profile>', 'aws profile name to use')
   .parse(process.argv);
 
+function getAWSCredentials() {
+  if (program.profile) {
+    return new AWS.SharedIniFileCredentials({ profile: program.profile });
+  }
+  if (process.env.AWS_ACCESS_KEY_ID && process.env.AWS_SECRET_ACCESS_KEY) {
+    return new AWS.Credentials({
+      accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+      secretAccessKey:
+        process.env.AWS_ACCESS_KEY_ID && process.env.AWS_SECRET_ACCESS_KEY,
+      sessionToken: process.env.AWS_SESSION_TOKEN,
```

Comment:
sessionToken will only work for those with MFA enabled, for people not running with MFA via SSM what will happen?

Reply:
This works for both as in this case sessionToken will be null which is default if not running MFA! Your comment did make me notice an uneeded && though unsure howit was working with it